src/AppBundle/Controller/AssetsController.php

```diff
@@ -53,11 +53,14 @@ public function assetAction($path, Request $request)
      *
      * @Method("GET")
      */
-    public function mapAction(string $latitude, string $longitude)
+    public function mapAction(Request $request, string $latitude, string $longitude)
     {
         $coordinates = new Coordinates($latitude, $longitude);
 
-        if (!$contents = $this->get('app.map.google_maps_static_provider')->get($coordinates)) {
+        $contents = $request->query->has('algolia')
```

Comment:
`$contents = $this->get('app.map.google_maps_static_provider')->get($coordinates, $request->query->has('algolia') ? '250x170' : '');`